src/t8_geometry/t8_geometry_implementations/t8_geometry_occ.cxx

```diff
@@ -652,11 +632,13 @@ t8_geometry_occ::t8_geom_evaluate_occ_hex (t8_cmesh_t cmesh, t8_gloidx_t gtreeid
        * The edges are indexed so that all edges which satisfy i_edge % 4 == 0 
        * have to multiplied with the inversed (1 - ref_coord) 
        * coordinate. All edges which satisfy i_edge % 4 == 1 have to multiplied with one 
-       * inversed ref_coord and so forth... */
+       * inversed ref_coord and so forth...
+       * An exception are edge 5 and 6, which have to be switched */
```

Comment:
Can you expand on that, do they have to be switched to match your approach of multiplying with `(1-ref_coord)` with the t8code enumeration?

Reply:
I tried my best